composer.json

```diff
@@ -30,16 +34,16 @@
     "require": {
         "php": "^7.1.3 || ^8.0",
         "monolog/monolog": "^1.6 || ^2.0",
-        "sentry/sentry": "^2.3.2"
+        "sentry/sentry": "^3.1"
     },
     "require-dev": {
         "coduo/php-matcher": "^3.2.2",
         "friendsofphp/php-cs-fixer": "^2.15",
-        "jangregor/phpstan-prophecy": "0.8.0",
-        "phpstan/phpstan": "0.12.25",
-        "phpstan/phpstan-phpunit": "0.12.8",
+        "jangregor/phpstan-prophecy": "^0.8.1",
```

Comment:
Packages were bumped in order to support PHP8, also constraints were changed to `^` because on each platform Composer will resolve them properly, no need to restrict to exact version.

Reply:
It was on purpose to avoid unwanted false positive on CI ^^

PHPStan updates may need code updates from version to version, even if it's less the case now.

Reply:
In terms of PHPStan you're using `0.*` while there's `1.7+` out there... It's work for another PR. But yeah, since it's only `require-dev` constraints can be closed.